locals.tf

```diff
@@ -1,140 +0,0 @@
-terraform {
-  experiments = [module_variable_optional_attrs]
```

Comment:
this is removed starting in v1.3.0 of Terraform and is the primary motivation of the PR